helmfile/values/networkpolicy-common.yaml.gotmpl

```
@@ -32,3 +34,12 @@ kured:
 
 velero:
   enabled: {{ and .Values.velero.enabled .Values.networkPolicies.velero.enabled }}
+
+certManager:
+  enabled: {{ .Values.networkPolicies.certManager.enabled }}
+  namespaces: {{ toYaml .Values.networkPolicies.certManager.namespaces | nindent 4 }}
+  {{- if eq .Environment.Name "workload_cluster" }}
+  namespaces: {{ toYaml .Values.user.namespaces | nindent 4 }}
+  {{- end }}
```

```suggestion
    {{- if eq .Environment.Name "workload_cluster" }}
    {{- toYaml .Values.user.namespaces | nindent 4 }}
    {{- end }}
```
I assume you want these to collapse into one right?